### PR TITLE
chore: add comment to explain origin of test

### PR DESCRIPTION
**Pull request by jschfflr**

### CODE REVIEW

**Comment by jschfflr** [comment]
@googlebot I consent.

**Comment by jschfflr** [comment]
@OrKoN Could you LGTM this change?

**Comment by jschfflr** [comment]
@googlebot I consent.

File: test/requestinterception.spec.ts [hunk 1]
```diff
@@ -52,13 +52,17 @@ describe('request interception', function () {
...
+    // @see https://github.com/puppeteer/puppeteer/pull/3105
     it('should work when POST is redirected with 302', async () => {
       const { page, server } = getTestState();
 
       server.setRedirect('/rredirect', '/empty.html');
       await page.goto(server.EMPTY_PAGE);
       await page.setRequestInterception(true);
-      page.on('request', (request) => request.continue());
+      page.on('request', (request) => {
+        console.log(request.method(), request.url());
```
Comment (OrKoN) [inline test/requestinterception.spec.ts]: why are console.log messages needed?

Comment (jschfflr) [reply test/requestinterception.spec.ts]: Good catch, thanks! I thought I had already removed it.

**Review by OrKoN — APPROVED** [review]

**Pull request merged**